web/mixins/editClient.js

```diff
@@ -0,0 +1,29 @@
+export default {
+  methods: {
+    async editClient(data) {
+      await this.$store
+        .dispatch("updateClient", data)
+        .then(() => {
+          this.$toasted.success("Client updated successfully", {
+            theme: "bubble",
+            position: "top-right",
+            duration: 2000,
+          });
+          this.$nextTick(() => {
```

Comment:
why have you use nextTick here ? Any issue facing here

Reply:
@ekta-kakadia removed